packages/create-cli/project.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "create-cli",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/create-cli/src",
+  "projectType": "application",
+  "implicitDependencies": ["nx-plugin"],
```

Comment:
The dependency on `nx-plugin` is already clear from imports, I don't see any need to add it as an implicit dependency too.

```suggestion
```